docker-compose.yml

```diff
@@ -49,6 +49,11 @@ services:
     image: srcd/superset:latest
     restart: unless-stopped
     environment:
+      ADMIN_LOGIN: admin
+      ADMIN_FIRST_NAME: admin
+      ADMIN_LAST_NAME: admin
+      ADMIN_EMAIL: admin@no-email.org
```

Comment:
I'm not familiar with `no-email.org`, maybe it's safer to use `admin@example.com`, since it's owned by IANA